--- a/venv/lib/python2.7/site-packages/pyelasticsearch/client.py	2019-02-06 13:37:53.332881000 -0500
+++ b/venv/lib/python2.7/site-packages/pyelasticsearch/client.py	2017-08-07 13:46:44.808186136 -0400
@@ -268,7 +268,7 @@
         # compatibility:
         try:
             # This implicitly converts dicts to JSON. Strings are left alone:
-            _, prepped_response = self._transport.perform_request(
+            prepped_response = self._transport.perform_request(
                 method,
                 path,
                 params=dict((k, self._utf8(self._to_query(v)))
@@ -295,6 +295,10 @@
               (error_message.startswith('IndexAlreadyExistsException') or
                'nested: IndexAlreadyExistsException' in error_message)):
             error_class = IndexAlreadyExistsError
+        elif (hasattr(error_message, 'startswith') and
+              (error_message.startswith('index_already_exists_exception') or
+               'nested: index_already_exists_exception' in error_message)):
+            error_class = IndexAlreadyExistsError
 
         raise error_class(status, error_message)
 
@@ -990,6 +994,26 @@
                                  query_params=query_params)
 
     @es_kwargs('ignore_unavailable')
+    def get_alias(self, index=None, alias='*', query_params=None):
+        """
+        Retrieve a listing of aliases
+
+        :arg index: The name of an index or an iterable of indices from which
+            to fetch aliases. If omitted, look in all indices.
+        :arg alias: The name of the alias to return or an iterable of them.
+            Wildcard * is supported. If this arg is omitted, return all aliases.
+
+        See `ES's indices-aliases API`_.
+
+        .. _`ES's indices-aliases API`:
+            http://www.elastic.co/guide/en/elasticsearch/reference/current/indices-aliases.html
+        """
+        return self.send_request(
+                'GET',
+                [self._concat(index), '_alias', self._concat(alias)],
+                 query_params=query_params)
+
+    @es_kwargs('ignore_unavailable')
     def get_aliases(self, index=None, alias='*', query_params=None):
         """
         Retrieve a listing of aliases
@@ -1206,7 +1230,7 @@
                                  [self._concat(index), '_optimize'],
                                  query_params=query_params)
 
-    @es_kwargs('level', 'wait_for_status', 'wait_for_relocating_shards',
+    @es_kwargs('level', 'wait_for_status', 'wait_for_no_relocating_shards',
                'wait_for_nodes', 'timeout')
     def health(self, index=None, query_params=None):
         """
--- a/venv/lib/python2.7/site-packages/pyelasticsearch-1.4.dist-info/METADATA	2019-02-06 13:37:53.336881000 -0500
+++ b/venv/lib/python2.7/site-packages/pyelasticsearch-1.4.dist-infoMETADATA	2019-02-06 13:41:50.567754590 -0500
@@ -22,7 +22,7 @@
 Classifier: Programming Language :: Python :: 3.4
 Classifier: Topic :: Internet :: WWW/HTTP :: Indexing/Search
 Requires-Dist: certifi
-Requires-Dist: elasticsearch (<2.0.0,>=1.3.0)
+Requires-Dist: elasticsearch (<6.0.0,>=1.3.0)
 Requires-Dist: urllib3 (<2.0,>=1.8)
 Requires-Dist: simplejson (>=3.0)
 Requires-Dist: six (<2.0,>=1.4.0)
